quarkus/pom.xml

```diff
@@ -105,6 +110,16 @@
       <artifactId>frontend</artifactId>
       <version>main</version>
     </dependency>
+    <dependency>
+      <groupId>com.redhat.openshift.knative.showcase</groupId>
+      <artifactId>cloudevents-pp-wasm</artifactId>
```

Comment:
Where's the source code for this dependency?

Reply:
The code is here: https://github.com/cardil/cloudevents-pretty-print (in Rust). It gets build onto a WASM module, and published on quay.io/cardil/cloudevents-pretty-print

The script in frontend app, downloads that WASM module from the registry, and packages it as jar onto local `~/.m2` directory.

That's done, so it is easily available for Java app (same as web files were provided before)